tests/unit/Endpoints/RefundsTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace PagarMe\Endpoints\Test;
+
+use PagarMe\Client;
+use PagarMe\Endpoints\Refunds;
+use PagarMe\Test\Endpoints\PagarMeTestCase;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\Psr7\Response;
+
+final class RefundsTest extends PagarMeTestCase
+{
+    public function refundsProvider()
```

Comment:
As we only have only a single test, maybe we don't need to use a data provider :wink: